packages/core/test/cross-chain-oracle/chain-adapters/Optimism_ChildMessenger.js

```diff
@@ -32,10 +32,10 @@ describe("Optimism_ChildMessenger", function () {
     const accounts = await hre.web3.eth.getAccounts();
     [l1Owner, controlledEOA, parentMessenger, rando] = accounts;
 
-    l2CrossDomainMessengerMock = await deployContractMock("OVM_L2CrossDomainMessenger", {
```

Comment:
and of course, all these were re-named in their official package. updating versions required us to change this.